src/containers/ConceptPage/conceptInterfaces.ts

```diff
@@ -23,7 +24,8 @@ export interface ConceptFormType extends ConceptType {
 
 export interface ConceptFormValues extends StrippedConceptType, FormValues {
   articles: ArticleType[];
-  conceptContent: string;
+  slatetitle: Value;
```

Comment:
Det er ikke mer ryddig om denne bare heter title? eller er det for å tydeliggjøre at den er gjort om til en slatevalue? Andre steder blir den vel fortsatt bare hetende `title` også etter `plainTextToEditorValue`, så krever flere sånne småendringer i så fall hvis vi skal ta i bruk FormikTitle i andre forms...

Reply:
jeg vet ikke. er bare slik det virket som om FormikTitle ville ha det. Så jeg antok de hadde en grunn til det

Reply:
@gunnarvelle ?

Reply:
åja trodde du lagde FormikTitle nå jeg! da er den vel brukt andre steder alt, så da er det vel lettere å bare beholde slatetitle

Reply:
Tittelfeltet for audio og image er ikkje slate-editorer, så det er for å skille mellom dei forskjellige title-felt-typene.